app/helpers/katello/katello_url_helper.rb

```diff
@@ -1,27 +1,19 @@
 module Katello
   module KatelloUrlHelper
     unless defined? CONSTANTS_DEFINED
-      FILEPREFIX = 'file'
-      PROTOCOLS = ['http', 'https', 'ftp', FILEPREFIX]
+      FILEPREFIX = ['file']
+      PROTOCOLS = ['http', 'https', 'ftp']
 
       CONSTANTS_DEFINED = true
     end
 
     def kurl_valid?(url)
-      valid_for_prefixes(url, PROTOCOLS)
-    end
-
-    def file_prefix?(url)
-      valid_for_prefixes(url, [FILEPREFIX])
-    end
-
-    private
-
-    def valid_for_prefixes(url, prefixes)
-      return false unless (scheme = URI.parse(url).scheme).present?
-      prefixes.include?(scheme.downcase)
+      return false if (scheme = URI.parse(url).scheme).blank?
+      return true if FILEPREFIX.include?(scheme.downcase)
+      URI.parse(url).host.present? && PROTOCOLS.include?(scheme.downcase)
```

Comment:
Does the absence of this code change break anything? Could we leave it alone until Katello#5664 is merged, before or after Rails4?